ehrql/tables/tpp.py

```diff
@@ -552,6 +553,160 @@ class clinical_events_ranges(EventFrame):
+
+
+    **Overview**
+
+    Antivirals and neutralising monoclonal antibodies (nMABs) for COVID-19 can be
+    administered in inpatient setting or, for outpatients, in COVID Medicine Delivery
+    Units (CMDUs) specifically set up for this purpose. For patients considered for
+    these treatments, clinicians submit completed forms to NHS England. Each row
+    represents one completed form for one course of treatment. Data received by
+    OpenSAFELY currently covers patients who were approved for treatment. The patient
+    may or may not have actually received the treatment or completed the course (but we
+    assume that they usually do). They may have another form completed for another
+    treatment, either because it was decided to give them a different treatment, or for
+    some other reason. They may in theory also have another form completed some months
+    later for another instance of infection.
+
+    Treatment dates may be in the past or future at the point when the form is
+    submitted.
+
+    Note that this dataset may contain **duplicate** rows – full duplicates are removed
+    but there may remain some partial duplicates.
+
+
+    **More Information**
+
+    * [Treatment guidelines](https://www.nice.org.uk/guidance/ta878)
+    * [Draft Data Report](https://docs.google.com/document/d/15o4x9sqHEO-sLm2dTqgm3PyAh72cdgOOmZC4AB3BTNk/) (currently only available to internal staff)
+    """
+
+    age_at_received_date = Series(
+        int,
+        description="""
+            Can occasionally be zero, presumably indicating an unknown or missing value
+            as minimum eligibility age is 12.
+        """,
+    )
+    count = Series(
+        int,
+        description="Number of forms.",
+    )
+    covid_indication = Series(
+        str,
+        description="Treatment setting/indication.",
+        constraints=[
+            Constraint.Categorical(
+                ["non_hospitalised", "hospitalised_with", "hospital_onset"]
+            )
+        ],
+    )
+    current_status = Series(
+        str,
+        description="Status of form/application.",
+        constraints=[
+            Constraint.Categorical(
+                [
+                    "Approved",
+                    "Treatment Complete",
+                    "Treatment Not Started",
+                    "Treatment Stopped",
+                ]
+            )
+        ],
+    )
+    diagnosis = Series(
+        str,
+        description="Always has the value 'Covid-19'.",
+    )
+    form_name = Series(
+        str,
+        description="""
+            Name and version of the patient registration form used to register the
+            treatment.
+        """,
+    )
+    intervention = Series(
+        str,
+        description="""
+            Intervention or therapeutic name. Expected to be one of:
+
+             * Paxlovid
+             * Sotrovimab
+             * Molnupiravir
+             * Remdesivir
+             * Casirivimab and imdevimab
```

Comment:
There is also now Sarilumab and Baricitinib
This should probably be a Categorial Constraint

Reply:
is `Sarilumab and Baricitinib` one entry on the list or two...?

Reply:
2; but there's another one that wasn't on the original list too (I've just checked). Full list is:
- Sotrovimab
- Molnupiravir 
- sarilumab (Note: all lower case in db, others are title case. I don't think it matters for querying because the collation is case insensitive)
- Baricitinib
- Paxlovid
- Remdesivir
- Tocilizumab
- Casirivimab and imdevimab

Reply:
The list for the raw table also needs updating

Reply:
Thanks, will update. Out of curiosity, where did you check that?

Reply:
ok, alphabetised & updated in both places.